### PR TITLE
Add `getting-started` docs to `next`

### PR DESCRIPTION
**Pull request by shawntabrizi**
Only thing missing here is "Next Steps" items, which we can add when we do final pass of the "next" documentation.

Uses _no_ scripts... although we might want to consider including the `getsubstrate.io` script since it is really nice for Mac/Linux users.

### CODE REVIEW

File: docs/getting-started.md [hunk 1]
```diff
@@ -2,4 +2,279 @@
...
+https://github.com/substrate-developer-hub/substrate-front-end-template
+```
+
+Install the node dependencies with:
+
+```bash
+yarn install
+```
+
+Run the front end with:
```
Comment (4meta5) [inline docs/getting-started.md]: For consistency, should we use front-end or front end? See line 184, "front-end"
```suggestion
Run the front-end with:
```

File: docs/getting-started.md [hunk 2]
```diff
@@ -2,4 +2,279 @@
...
+
+```bash
+yarn start
+```
+
+Connect to the front-end at [`localhost:8000`](http://localhost:8000).
+
+> **Note:** You need to have a local Substrate node running to interact with this UI.
```
Comment (4meta5) [inline docs/getting-started.md]: Would be nice to link here to direction for getting a local Substrate node running.

File: docs/getting-started.md [hunk 3]
```diff
@@ -2,4 +2,279 @@
...
+
+## Interact with Substrate
+
+The fastest way to interact any local or public Substrate network it to visit Polkadot-JS Apps:
```
Comment (4meta5) [inline docs/getting-started.md]: > 
```suggestion
The fastest way to interact any local or public Substrate network is to visit Polkadot-JS Apps:
```

File: docs/getting-started.md (hunk 1 above)
Comment (shawntabrizi) [reply docs/getting-started.md]: "front-end" is an adjective:
> front-end template

Describes the kind of template.

"front end" is a noun, as it is being used here.

**Comment by shawntabrizi** [comment]
@4meta5 Nits addressed :)

**Pull request merged**